%% The following is a directive for TeXShop to indicate the main file
%%!TEX root = diss.tex

\chapter{Abstract}

Genomic analyses of tumours can not only reveal actionable somatic mutations, but also germline variants with clinical implications for patients and their families. While sequencing of tumour-normal pairs would enable simultaneous identification of clinically important germline variants, matched normal samples are often not obtained in clinical practice. Furthermore, tumour specimens are typically formalin-fixed paraffin-embedded (FFPE), which induces DNA damage that poses challenges in molecular testing. A framework that leverages clinical tumour sequencing for germline testing is cost-saving because only patients with potential germline variants would be referred to downstream confirmatory testing. However, this would require evaluating usability of FFPE DNA for germline testing and establishing an approach to distinguish between germline and somatic variants in tumour-only analyses. We retrospectively analyzed clinical amplicon sequencing data from 213 patients with tumour and matched normal samples. To evaluate the usability of FFPE DNA for germline testing, we characterized formalin-induced DNA damage by comparing efficiency in amplicon enrichment and sequencing results of FFPE DNA to blood, the gold standard for germline testing. Although formalin-induced DNA damage including fragmentation and cytosine deamination were detectable, we determined that these discrepancies were either technically negligible or could be minimized using appropriate methods. We also found that 93.8\% of germline alterations identified in blood were present with the same allelic statuses in FFPE tumours. This implies that the majority of germline genetic changes are retained in the tumour genome, demonstrating the reliability of using tumour DNA for germline variant calling. Lastly, we assessed the application of variant allele frequency (VAF) threshold to delineate germline and somatic variants in tumour-only analyses. We reported that a VAF cut-off of 30\% would correctly identify 94\% of germline alterations, while erroneously submit 10\% of false positives, which are somatic mutations, for follow-up germline testing. This underscores the high sensitivity and precision of using VAF threshold to discriminate between germline and somatic variants in tumour-only analyses. Our results collectively demonstrate an invaluable finding in clinical genomics wherein leveraging FFPE tumour sequencing for identification of germline variants could be a practical, cost-efficient approach for providing germline testing. 
