%% The following is a directive for TeXShop to indicate the main file
%%!TEX root = diss.tex

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\chapter{Discussion}
\label{ch:Discussion}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Clinical tumour sequencing can inform medical decision-making for cancer patients. While sequencing of tumour-normal pairs would enable accurate identification of somatic mutations and simultaneous detection of clinically important germline variants, matched normal samples are often not obtained in clinical practice. Genomic analyses of tumours can reveal actionable somatic mutations and clinically relevant germline alterations \cite{Meric-Bernstam2016, Schrader2015, Jones2015a}. However, tumour tissues are commonly FFPE, which represents a challenge in clinical genomics. Formalin fixation damages nucleic acid through fragmentation and cytosine deamination, affecting molecular testing with FFPE DNA \cite{Do2015a, Kim2017, Ofner2017, Oh2015, Wong2013, Wong2014, Sikorsky2007}. Hence, usability of FFPE DNA for germline testing and approaches to discriminate between germline and somatic variants in tumour-only analyses must be evaluated. These assessments would facilitate optimization of workflows to identify potential germline alterations using clinical tumour sequencing.

In this study, we retrospectively analyzed targeted sequencing data from tumour and matched blood specimens of 213 cancer patients. Our findings were consistent with DNA fragmentation and cytosine deamination being common forms of DNA damage in FFPE specimens. While the impact of formalin fixation on amplicon enrichment and sequencing results was detectable, we determined that these discrepancies were minor and could be minimized using methods such as using shorter amplicons and enriching for longer DNA templates. We also found that 98.0\% of germline alterations identified in the blood using our panel test were present in the FFPE tumours. This implies that a high proportion of germline genetic changes were retained in the tumour genome, demonstrating the reliability of using tumour DNA for germline variant calling. Finally, we assessed the application of VAF thresholds to delineate germline and somatic variants in tumour-only analyses. We reported that a VAF cut-off of 15% would correctly identify 99% of germline alterations in FFPE tumours, but erroneously submit 14% of somatic mutations (false positives) for follow-up germline testing. Because our gene panel and patient cohort were relatively small, we were only able to identify germline variants that were predictive of drug response. Nevertheless, we surmised that the VAF cut-off approach could be extended to predict the statuses of pathogenic germline variants such as alterations in \textit{BRCA} genes.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Formalin-induced DNA damage has minor effects on sequencing metrics}
\label{sec:Formalin-inducedDNAdamagehasminoreffectsonsequencingmetrics}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Several studies have reported findings that are consistent with our assessment of formalin-induced DNA damage in FFPE specimens. To assess the usability of FFPE DNA for germline testing, we compared efficiency in amplicon enrichment and sequencing results of FFPE DNA to blood, which is a gold standard for germline testing. We noted lower efficiency in amplicon enrichment in FFPE DNA, with a more pronounced decrease in coverage depth for longer amplicons in the panel. Similarly, Shi et al. \cite{Shi2002}, Didelot et al. \cite{Didelot2013}, and Wong et al. \cite{Wong2013} demonstrated that shorter amplicons gave rise to better PCR amplification success in FFPE DNA, indicating the presence of fragmentation damage, which yielded template DNA of shorter fragment lengths. While we observed comparable proportion of on-target aligned reads between FFPE and blood DNA, there were minor discrepancies in coverage depth and uniformity of target bases in FFPE DNA. Various groups have also reported disparities in coverage depth and uniformity in FFPE DNA when compared to DNA extracted from either fresh frozen or unfixed specimens \cite{Wong2013, Betge2015, Spencer2013}. Additionally, Wong et al. \cite{Wong2014} and Didelot et al \cite{Didelot2013} showed inverse correlations between coverage depth and the degree of DNA fragmentation in FFPE DNA, suggesting that formalin-induced fragmentation damage could be accountable for such discrepancies in sequencing results. Although we detected differences in sequencing results between FFPE and blood DNA, we concluded that these effects were either minor or statistically insignificant. As for the discrepancy in amplicon enrichment, shorter amplicons can be designed to circumvent the drawback of fragmentation damage in FFPE samples.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Sequence artifacts induced by cytosine deamination tend to occur at low allelic frequency}
\label{sec:Sequenceartifactsinducedbycytosinedeaminationtendtooccuratlowallelicfrequency}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Cytosine deamination is a major cause of sequence artifacts in formalin-fixed specimens \cite{Wong2014, Do2012, Oh2015, Spencer2013, Do2013, Kim2017, Chen2014}. Herein, we observed increased C$>$T/G$>$A artifacts in FFPE DNA compared to blood. Artifactual C$>$T/G$>$A changes are formed by incorporation of adenines in the complementary DNA strand at uracil lesions generated by deamination of cytosines \cite{Do2015a}. When measuring frequency of sequence artifacts at different allele frequency ranges, Wong et al. \cite{Wong2014} reported higher C$>$T/G$>$A transitions at a lower allele frequency range (1--10\% \textit{vs.} 10--25\%). This finding led us to compare the fraction of base changes at different allele frequency ranges, including 1--10\%, 10--20\%, and 20--30\%. Indeed, we observed a substantial increase in C$>$T/G$>$A within the 1--10\% allele frequency range. However, we were unable to separate FFPE artifacts from low-allelic-fraction somatic mutations within the different allele frequency ranges due to the lack of matched fresh frozen or unfixed tumour tissues. Somatic mutations can occur at VAFs that deviate significantly from a diploid zygosity (i.e. heterozygous variant should have VAF close to 50\%, whereas homozygous variant should have VAF close to 100\%) because of low tumour content or tumour heterogeneity \cite{Kim2017a, Xu2017, Carrot-Zhang2016, Tian2015, Cai2016}. Therefore, further workflow optimization should be performed for the purpose of identifying clinically relevant somatic mutations in the tumour genome. A method to reduce sequence artifacts caused by cytosine deamination is treatment with uracil-DNA glycosylase (UDG) before sequencing. UDG is an enzyme capable of depleting uracil lesions in DNA, giving rise to abasic sites. During PCR amplification, cytosine bases are restored at abasic sites by using the complementary DNA strand as template, which consists of guanine bases opposite of the uracil lesions \cite{Do2015a}. Several studies showed that pre-treatment of FFPE DNA with UDG can markedly reduce C$>$T/G$>$A sequence artifacts \cite{Do2013, Kim2017, Do2012}. However, this approach cannot correct sequence artifacts at CpG dinucleotides because these cytosines are typically methylated, and deamination of 5-methyl cytosines generates thymines instead of uracil bases, which are resistant to UDG repair \cite{Do2013}.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Sequence artifacts other than those caused by cytosine deamination are detected}
\label{sec:Sequenceartifactsotherthanthosecausedbycytosinedeaminationaredetected}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

We also observed elevated levels of A$>$G/T$>$C artifacts in FFPE DNA, albeit to a lesser extent compared to C$>$T/G$>$A artifacts. Likewise, Wong et al. \cite{Wong1998} reported that 35\% of sequence artifacts in Sanger sequencing of the \textit{BRCA1} gene were A$>$G/T$>$C nucleotide changes. We speculate that increase in A$>$G/T$>$C artifacts is caused by deamination of adenine to generate hypoxanthine, which forms base pairs with cytosine instead of thymine. This results in transformation of A-T base pairs to G-C base pairs. Deamination of adenine to hypoxanthine can be catalyzed by an acidic environment \cite{Wang2010}, which can arise in FFPE specimens because formaldehyde can be oxidized to generate formic acid \cite{Do2015a}.

Acidic conditions also promotes depurination, creating abasic sites. Many DNA polymerases selectively incorporate adenines across abasic sites, while guanines and small deletions are integrated in fewer cases \cite{Heyn2010}. Despite being statistically insignificant, we observed a subset of FFPE specimens with higher fractions of C$>$A/G$>$T artifacts. These artifactual changes could have resulted from depurination of guanines, followed by incorporation of adenines by DNA polymerase in the complementary strand, which alters G-C base pairs to A-T base pairs. Heyn et al. \cite{Heyn2010} reported that DNA polymerases demonstrated varying bypass rates at abasic sites. For instance, AmpliTaq Gold, \textit{Pfu}, and Platinum Taq HiFi extended across lower frequency of abasic sites compared to Platinum Taq, \textit{Bst} and \textit{Sso}-Dpo4 ($<$34\% \textit{vs.} $>$77\%) \cite{Heyn2010}. Thus, selection of a high fidelity DNA polymerase could lessen these forms of sequence artifacts.

Costello et al. \cite{Costello2013} discovered that C$>$A/G$>$T artifacts can also occur due to oxidation of DNA during the shearing process, converting guanines to 8-oxoguanine lesions. This conversion is highly dependent on the surrounding 5' and 3' bases of the guanine, in which guanines within GGC are the most susceptible to oxidation. 8-oxoguanine can form base pairs with cytosine and adenine, and mispairing with adenine would give rise to artifactual C$>$A/G$>$T transversions. However, this was not the cause of C$>$A/G$>$T artifacts in our data because both blood and FFPE DNA were sheared, and we did not observe simultaneous C$>$A/G$>$T increments in both specimen types compared to other types of base changes.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Storage time of FFPE blocks correlates with the extent of formalin-induced DNA damage}
\label{sec:StoragetimeofFFPEblockscorrelateswiththeextentofformalin-inducedDNAdamage}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Ludyga et al. \cite{Ludyga2012} demonstrated that long-term storage of FFPE blocks led to increased DNA fragmentation, producing shorter template DNA for PCR amplification. Furthermore, Carrick et al. \cite{Carrick2015} showed that increased storage time of FFPE blocks affects sequencing coverage and depth in NGS data. These findings are in agreement with our results, in which we found negative associations between age of paraffin blocks and efficiency in amplicon enrichment, coverage depth of target bases, and percentage of on-target aligned reads. As well, we observed a positive correlation between age of paraffin blocks and fraction of C$>$T/G$>$A artifacts, an outcome of stochastic enrichment. Due to exposure to environmental conditions, older FFPE blocks tend to produce increasingly fragmented DNA, which results in lower amounts of amplifiable DNA. Consequently, there is a higher chance of amplifying template DNA with sequence artifacts caused by formalin, yielding increased frequency of artifactual nucleotide changes in older FFPE specimens \cite{Wong2014}. These results demonstrating the correlations between storage time of paraffin blocks and sequencing variables suggest that if multiple FFPE blocks are available, the specimen with the shorter storage time should be selected for molecular testing. However, clinical specimens are often limited, making sample selection a rare option in the diagnostic setting. As such, other approaches to eliminate sequence artifacts should be considered such as application of molecular barcodes and hybridization-capture enrichment, which allow tracking of DNA templates \cite{Eijkelenboom2016, Samorodnitsky2015, Peng2015, Wong2013}. This would enable detection of variants that are only supported by the same template DNA, indicating a higher chance that these variants are sequence artifacts and should be interpreted with caution.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Germline variants are highly retained in the tumour genome}
\label{sec:Germlinevariantsarehighlyretainedinthetumourgenome}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Various groups have identified clinically significant germline alterations through analyzing tumour genomes \cite{Schrader2015, Meric-Bernstam2016, Jones2015a, WcWhinney2009}. Schrader et al. \cite{Schrader2015} reported that potential pathogenic germline variants in cancer-predisposing genes were conserved in the tumours of 91.9\% of patients in their study cohort (182 of 198 patients), whereas 21.4\% of these patients (39 of 182 patients) demonstrated LOH or other forms of mutations in the remaining wild type allele. We found that 98.0\% (1981$/$2022) of germline alterations identified in the blood were retained in the tumour, a finding that is in line with previous work. This suggests that tumour DNA could be a reliable source for detecting germline alterations, implying that a tumour-only sequencing protocol could be leveraged for pre-screening of germline variants before submission to downstream confirmatory testing. A framework as such could provide germline testing in a time- and cost-effective manner because only selected patients (i.e. those with potential germline alterations that are clinically important) would require follow-up.

Out of the 98.0\% of germline alterations that were retained in the tumours, 4.3\% (85$/$1981) of variants demonstrated discordant allelic status between blood and tumour. We also identified discordant germline variants, which were present in the blood but not detected in the tumour, that were caused by LOH (heterozygous variant in the blood but wild type in the tumour; 34$/$41 variants) and low sequencing coverage ($<$ 100x; 7$/$41 variants). All tumour specimens in our study were formalin-fixed, therefore it is possible that DNA damage induced by formaldehyde exposure played a role in creating discordant germline variants. Variant discordance can also be caused by mutagenesis in the tumour, such as somatic CNVs in the region of the germline variant. For instance, Gross et al. \cite{Gross2013} showed a high prevalence of \textit{DPYD} CNVs in high-grade triple negative breast cancer, particularly in cases with copy number loss of the \textit{BRCA1} DNA-repair gene. The common fragile site FRA1E is located within the \textit{DPYD} gene and its stability is highly dependent on intact \textit{BRCA1} \cite{Arlt2004}. Hence, deficiency in \textit{BRCA1} protein would result in increased fragility of FRA1E, leading to genomic rearrangements in \textit{DPYD}. As germline variants in the \textit{DPYD} gene can predict susceptibility to 5-FU-related toxicity, somatic CNVs in \textit{DPYD} could affect the detection of these germline variants in tumour genomic sequencing.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{The use of VAF thresholds is feasible for distinguishing between germline and somatic alterations in tumour-only analyses}
\label{sec:TheuseofVAFthresholdsisfeasiblefordistinguishingbetweengermlineandsomaticalterationsintumour-onlyanalyses}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Although sequencing of tumour-normal pairs would enable accurate identification of germline and somatic variants, this approach is not routinely practiced in clinical genomics due to inadequate funding and facilities to store additional specimens. Methods to distinguish between germline and somatic alterations in tumour-only analyses have been described by different groups \cite{Hiltemann2015, Jones2015a, Garofalo2016}. Hiltemann et al. \cite{Hiltemann2015} used a virtual normal that was assembled by aggregating whole-genome-sequenced normal samples from 931 healthy and unrelated individuals, whereas Jones et al. \cite{Jones2015a} resorted to using an unmatched normal sample and public databases such as dbSNP, 1000 Genomes Project, and COSMIC, as well as effect prediction tools. We leveraged the fact that the VAFs of somatic mutations typically deviate from 50\% and 100\% for heterozygous and homozygous variants, respectively, and employed VAF thresholds to differentiate between variant statuses. Our approach managed to achieve high sensitivity and precision, therefore verifying the feasibility of using VAF threshold to differentiate between germline and somatic alterations in the absence of matched normal samples.

The VAF threshold method takes advantage of genetic impurity and heterogeneity of tumours, which render the deviation of somatic VAFs from diploid zygosity. Jones et al. \cite{Jones2015a} discovered that performance of the VAF threshold approach was highly dependent on tumour purity. While the use of VAF can correctly identify all germline and somatic alterations in tumours with $<$ 50\% purity, this accuracy was not observed for specimens with higher tumour content. In fact, only 12.5\% of cancer-predisposing germline variants and an average of 48\% of somatic mutations were accurately predicted using the VAF method in tumours with more than 50\% tumour content \cite{Jones2015a}. Unfortunately, pathologic estimation of tumour content was not available for our analyses. However, we speculate that the tumour specimens in our data set are highly impure or heterogeneous, thereby contributing to the high sensitivity and precision attained by the VAF threshold approach. While there are bioinformatic algorithms available to infer clonality and impurity estimates of tumours, many of these methods require matched normal controls or are not compatible with targeted sequencing data \cite{Yadav2015}. Nevertheless, this information should be integrated into clinical pipelines to enhance the performance of using a VAF threshold approach to distinguish between germline and somatic alterations in tumour-only genomic analyses.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Limitations and future directions}
\label{sec:Limitationsandfuturedirections}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

There are several limitations in our study. First, we did not manually review every single variant called by our pipeline. Only variants located within primer regions were manually inspected, while our variant filter also included common artifacts that were curated during clinical assessment. Hence, it is highly possible that sequence artifacts are present in our data set, particularly low-allelic-fraction variants (i.e. $<$ 10\%) detected in the blood. Variant inspection using a genome browser is routinely conducted by genomic analysts in clinical practice to decrease the risk of reporting false positive results \cite{Strom2016, Garofalo2016}. However, manual review of variants was not implemented in our study because our analyses were focused on evaluating analytical validity instead of producing genomic information for clinical decision-making. Moreover, the large number of variants in our study would be time-consuming and unfeasible for manual inspection. Our evaluation of the VAF threshold approach in differentiating between germline and somatic variants is favourable of the framework to implement initial screening for germline variants in clinical tumour sequencing before follow-up germline testing.

The small gene panel and patient cohort size are caveats in our study. Although we were able to identify germline variants that can influence drug response, we did not report any pathogenic germline variants that were associated with cancer predisposition in our data set. Hence, we can only speculate that our approach could be applicable to variants in cancer-predisposing genes. Studies that were able to identify pathogenic germline variants were performed with cohort sizes and gene panels that were substantially larger than this study. For instance, the study by Schrader et al. \cite{Schrader2015}, which revealed pathogenic germline variants in 16\% of patients, was performed in a cohort of 1566 patients and 341 genes were screened. To determine whether the VAF threshold method can be applied to detect genetic alterations linked to cancer susceptibility, further assessment involving a larger patient cohort and surveying known cancer-predisposing genes should be carried out.

The present study addresses two problems faced by using tumour genomic sequencing to identify germline alterations: the widespread use of FFPE tumours and the challenge in differentiating between germline and somatic variants in tumour genomes. Archival FFPE tissues remain a sizable resource for cancer genomic studies and clinical genomic sequencing. Thus, there is a need to understand the extent of the different forms of DNA damage induced by formalin. Our analyses not only provided insights on the impact of formalin-induced DNA damage on amplicon-based NGS data, but also helped us devised guidelines to minimize these effects. Formalin fixation followed by paraffin embedding is an attractive method to preserve tissue morphology for histologic assessment because it allows storage at ambient temperature, which reduces cost incurred by maintaining freezers required for fresh-frozen samples. Yet, many studies, including ours, have indicated the side effects of formaldehyde exposure on nucleic acid \cite{Do2015a, Kim2017, Ofner2017, Oh2015, Wong2013, Wong2014, Sikorsky2007}. Instead of investing efforts into mitigating these side effects, a potential solution is to transition from the use of formalin to the UMFIX (Sakura Finetek USA, Inc.) fixative, which is capable of preserving both cellular morphology for pathologic review and macromolecules, including DNA \cite{Vincek2003}.

Most clinical laboratories conduct tumour-only sequencing and apply approaches to distinguish between germline and somatic alterations. Without matched normal samples, interpretation of variants becomes complicated. Jones et al. \cite{Jones2015a} and Garofalo et al. \cite{Garofalo2016} concluded that sequencing of tumour-normal pairs is the best practice to accurately identify variant statuses. For a center to provide tumour-normal paired sequencing, it must be equipped to collect, analyze, and report germline findings. This includes establishing appropriate pre-test and post-test counseling, protocols to secure patient consent and manage variant of uncertain significance, and frameworks to communicate results that may implicate the patients' relatives. While various groups recommend the sequencing of tumour-normal pairs, some centers simply do not have the funding or infrastructure to implement this as a standard practice. Furthermore, the American College of Medical Genetics and Genomics (\acs{ACMG}) recommended that clinical laboratories report incidental variants in 56 genes that are associated with disease risk in DNA derived from germline samples, including matched normal samples that only serve the purpose of subtracting germline variants to identify somatic mutations in tumours \cite{Green2013}. Interrogation of these genes suggested by the ACMG guidelines could result in detection of more variants with uncertain significance, which might pose more harm than good to patients. Additionally, cases in which only FFPE tumour blocks exist for a deceased patient would greatly benefit from approaches in differentiating between germline and somatic variants. For example, if the deceased individual is suspected to be a carrier of an inheritable disease, the ability to accurately identify the germline risk allele could prompt germline testing for the individual's relatives and facilitate preventive care. Thus, establishing approaches to tell apart germline and somatic variants in tumour genomic analyses still has its advantages from a clinical and financial perspective.

To summarize, we showed that the common forms of formalin-induced DNA damage in tumour samples were DNA fragmentation and cytosine deamination. Because these effects were either minor or statistically insignificant compared to DNA extracted from blood, this justified the feasibility of using FFPE DNA for germline testing. Characterization of formalin-induced DNA damage could also assist in establishing recommendations to enhance amplicon enrichment and sequencing results. We also reported a high retention rate of germline alterations in the tumour genomes, suggesting the reliability of using tumour DNA for germline variant calling. Finally, we showed that application of VAF thresholds can achieve high sensitivity and precision in distinguishing germline alterations from somatic mutations in tumour-only analyses. This supports the framework of leveraging clinical tumour sequencing for identification of germline alterations. Subsequently, only patients with potential germline variants are referred to follow-up testing. A framework in which selected patients are referred to downstream confirmatory testing represents a time- and cost-effective approach to deliver germline testing.

\endinput
Nevertheless, extrapolation of this approach for discriminating between germline and somatic variants in cancer-predisposing genes needs further evaluation.

These potentially artifactual variants account for 6\% of all germline variants identified in blood DNA, thereby compromising sensitivity of the VAF threshold method.

Considering that our goal was to predict germline status using tumour DNA, disproportionate base changes between FFPE and blood DNA within these allele frequency ranges suggest that germline calls should be made at $>$ 30\% VAF to avoid false positives that could either arise from true somatic mutations or FFPE artifacts.
