%% The following is a directive for TeXShop to indicate the main file
%%!TEX root = diss.tex

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\chapter{Discussion}
\label{ch:Discussion}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Genomic analyses of tumours can reveal druggable somatic mutations, as well as clinically relevant germline alterations that are beneficial to patients and their families \cite{Meric-Bernstam2016, Schrader2015, Jones2015a}. While sequencing of tumour-normal pairs can enable differentiation between germline and somatic variants, matched normal samples are often not obtained in the clinical setting. Moreover, FFPE tumour tissues represent another challenge in clinical genomics. Formalin fixation damages nucleic acid through fragmentation and cytosine deamination, which affect molecular testing with FFPE DNA \cite{Do2015a, Kim2017, Ofner2017, Oh2015, Wong2013, Wong2014, Sikorsky2007}. Hence, usability of FFPE DNA for germline testing and approaches to discriminate between germline and somatic variants in tumour-only analyses must be evaluated. These assessments would facilitate optimization of workflows to identify potential germline alterations using clinical tumour sequencing.

In this study, we retrospectively analyzed targeted sequencing data from tumour and matched blood specimens of 213 cancer patients. Our findings demonstrated that DNA fragmentation and cytosine deamination were common forms of DNA damage in FFPE specimens. While the impact of formalin fixation on amplicon enrichment and sequencing results was detectable, we determined that these discrepancies were either technically negligible or could be minimized using appropriate methods. We also found that the majority of germline alterations identified in blood using our panel test were present with the same allelic statuses in FFPE tumours. This implies that a high proportion of germline genetic changes is retained in the tumour genome, demonstrating the feasibility of using tumour DNA for germline variant calling. Finally, we assessed the application of VAF threshold to delineate germline and somatic variants in tumour-only analyses. We reported that a VAF cut-off of 30\% would correctly identify 94\% of germline alterations, while erroneously submit 10\% of false positives, which are somatic mutations, for follow-up germline testing. Because our gene panel and patient cohort are relatively small, we were only able to identify germline variants that are predictive of drug response. However, we surmised that application of this VAF cut-off could be expanded to predict the statuses of pathogenic germline variants such as alterations in \textit{BRCA} genes.

Several studies have reported findings that are consistent with our assessment of formalin-induced DNA damage in FFPE specimens. To assess the usability of FFPE DNA for germline testing, we compared efficiency in amplicon enrichment and sequencing results of FFPE DNA to blood, which is a gold standard for germline testing. We noted lower efficiency in amplicon enrichment in FFPE DNA, with a more pronounced decrease in coverage depth for longer amplicons in the panel. Similarly, Shi et al\cite{Shi2002}, Didelot et al\cite{Didelot2013}, and Wong et al\cite{Wong2013} demonstrated that shorter amplicons gave rise to better PCR amplification success in FFPE DNA, indicating the presence of fragmentation damage, which yields template DNA of shorter fragment lengths. While we observed comparable proportion of on-target aligned reads between FFPE and blood DNA, there were minor discrepancies in coverage depth and uniformity of target bases in FFPE DNA. Various groups have also reported disparities in coverage depth and uniformity in FFPE DNA when compared to DNA extracted from either fresh frozen or unfixed specimens \cite{Wong2013, Betge2015, Spencer2013}. Additionally, Wong et al \cite{Wong2014} and Didelot et al \cite{Didelot2013} showed inverse correlations between coverage depth and the degree of DNA fragmentation in FFPE DNA, suggesting that formalin-induced fragmentation damage could be accountable for such discrepancies in sequencing results. Although we detected differences in sequencing results between FFPE and blood DNA, we concluded that these effects were minor and technically insignificant. As for the discrepancy in amplicon enrichment, shorter amplicons should be designed to circumvent the drawback of fragmentation damage in FFPE samples.

Cytosine deamination is the major cause of sequence artifacts in formalin-fixed specimens \cite{Wong2014, Do2012, Oh2015, Spencer2013, Do2013, Kim2017, Chen2014}. Herein, we observed increased C$>$T/G$>$A artifacts in FFPE DNA compared to blood. Formation of artifactual C$>$T/G$>$A changes is caused by incorporation of adenine in the complementary DNA strand during PCR amplification at uracil lesions that are resulted by deamination of cytosines \cite{Do2015a}. When measuring frequency of sequence artifacts at different allele frequency ranges, Wong et al \cite{Wong2014} reported higher C$>$T/G$>$A transitions at a lower allele frequency range (1--10\% \textit{vs.} 10--25\%). This finding led us to compare the fraction of base changes at allele frequency ranges, including 1--10\%, 10--20\%, and 20--30\%. Likewise, we observed a substantial increase in C$>$T/G$>$A within the 1--10\% allele frequency range. We were unable to separate FFPE artifacts from low frequency somatic mutations within these allele frequency ranges because of the lack of matched fresh frozen tumour tissues. As our goal is to predict germline status, disproportionate base changes between FFPE and blood DNA within these allele frequency ranges suggest that germline call should be made at $>$ 30\% VAF to avoid false positives that could either arise from true somatic mutations or FFPE artifacts. Nevertheless, somatic mutations can occur at VAFs that deviate significantly from a diploid zygosity due to tumour admixture samples, meaning that. Therefore, genetic alterations at $<$ 30\% VAF are clinically significant and further optimization of protocols must be performed to subtract FFPE artifacts from true somatic mutations. One method to reduce sequence artifacts as a result of cytosine deamination is through treatment with UDG

Furthermore, we observed increased C$>$T/G$>$A base transitions in FFPE DNA, which are caused by cytosine deamination, a predominant type of sequence artifact induced by formalin. We also observed differences in frequency of base changes between FFPE and blood DNA within the allele frequency ranges of 1--10\%, 10--20\%, and 20--30\%, implying high prevalence of FFPE artifacts or somatic mutations within these allele frequency ranges. Based on this result, it is recommended that germline calls are made at $>$ 30\% VAF to avoid false positives arising from true somatic mutations or FFPE artifacts. Additionally, we showed that increased storage time of FFPE blocks resulted in poorer amplicon enrichment and sequencing results, as well as elevated levels of C$>$T/G$>$A artifacts. Therefore, if multiple FFPE specimens are available, the paraffin block with the shorter storage time should be selected for genomic testing. Overall, discrepancies in enrichment efficiency and sequencing results between FFPE and blood DNA were detectable in our dataset. However, we found that these differences were technically negligible or can be minimized using appropriate methods, thereby confirming the feasibility of using FFPE DNA for germline testing.

Furthermore, we showed elevated levels of C$>$T/G$>$A base changes in FFPE specimens compared to blood. This finding is consistent with several studies demonstrating cytosine deamination as the main source of sequence artifact in formalin-fixed specimens. In addition to C$>$T/G$>$A base changes, we observed increased A$>$G/T$>$C in FFPE DNA, although this difference was smaller than C$>$T/G$>$A. Our findings also demonstrated that base changes at the 1-10\%, 10-20\%, and 20-30\% allele frequency ranges were more prevalent in FFPE DNA than blood. This result implies that germline calls should be made at $>$30\% VAF to remove the majority of formalin-induced artifactual changes and somatic mutations. Finally, we showed that sequencing metrics are dependent on storage time of paraffin blocks. Older FFPE blocks were more likely to yield lower efficiency in amplicon enrichment, leading to poorer sequencing results and increased prevalence of artifactual C>T/G>A transitions. Thus, this indicates that if multiple FFPE specimens are available, the specimen with the lower storage time should be selected for molecular analysis.




Oh \textit{et al.}\cite{Oh2015} and Spencer \textit{et al.}\cite{Spencer2013} reported shorter insert sizes in FFPE DNA that were subjected to whole exome and hybridization capture sequencing, respectively. Additionally, Didelot \textit{et al.}\cite{Didelot2013} and Wong \textit{et al.}\cite{Wong2013} examined the integrity of DNA isolated from FFPE samples and demonstrated that FFPE DNA have shorter fragment lengths.

We observed lower efficiency in amplicon enrichment in FFPE DNA, with a more pronounced decrease in coverage depth for longer amplicons in the panel. While we noted comparable proportion of on-target aligned reads between FFPE and blood DNA, there were minor discrepancies in coverage depth and uniformity of target bases in FFPE DNA. These results are consistent with several studies reporting extensive fragmentation of DNA extracted from FFPE samples \cite{Wong2014, Didelot2013, Oh2015, Wong2013, Betge2015, Spencer2013}.

Various studies also showed that increased fragmentation damage in FFPE DNA is associated with low coverage depth and uniformity, which is in agreement with our findings \cite{Wong2013, Wong2014, Didelot2013, Betge2015, Spencer2013}. For instance, Betge \textit{et al.}\cite{Betge2015} examined the amount of template DNA in FFPE samples using a qPCR assay and found an inverse correlation between amount of template DNA and coverage depth. Although our findings demonstrated lower efficiency in amplicon enrichment of FFPE DNA, this disparity only led to negligible effects in coverage depth and uniformity. Nevertheless, shorter amplicons could be designed for the panel test to ensure improved enrichment and sequencing outcomes of FFPE DNA.

To assess the usability of FFPE DNA for germline testing, we compared efficiency in amplicon enrichment and sequencing results of FFPE DNA to blood, which is a gold standard for germline testing. Our findings demonstrated that DNA fragmentation and cytosine deamination were common forms of DNA damage in FFPE specimens. We noted lower efficiency in amplicon enrichment in FFPE DNA, with a more pronounced reduction in coverage depth for longer amplicons in the panel. This suggests that the use of shorter amplicons could achieve improved enrichment and sequencing coverage, mitigating formalin-induced fragmentation effects.

Our findings were consistent with several studies reporting the effects of DNA fragmentation

something about yes we noticed lower coverage depth too

This is which is evident of fragmentation damage in FFPE DNA, which leads to shorter fragment sizes that are not amenable to PCR amplification for longer amplicons, suggesting that

A key constraint in clinical genomics is the lack of matched normal samples to enable discrimination between germline and somatic alterations in tumour-only analyses. Analysis of variant concordance between



A key concern of using FFPE DNA for clinical genomic testing is the prevalence of sequence artifacts caused by cytosine deamination.

other forms of sequence artifacts

Our findings confirmed that DNA fragmentation and cytosine deamination were common forms of DNA damage in FFPE specimens.

age of paraffin blocks

can facilitate optimization of diagnostic workflow.

concordance, how reliable is the tumour DNA for germline testing

approach using variant allele frequency

Conclusion: As MPS becomes increasingly incorporated into clinical diagnostic work- flows, it is important to assess DNA damage caused by formalin fixation, as this will greatly optimise diagnostic workflows, increase accuracy of results and lead to better outcomes for patients.

This enabled characterization of formalin-induced DNA damage in our data and its impact on germline variant calling in FFPE DNA.

We observed lower , which can be caused by reduced template DNA as a result of formalin-induced DNA fragmentation. Although FFPE and blood DNA managed to achieve comparable proportions of on-target read alignments, which are aligned reads used for variant calling, our results showed minor discrepancies in coverage depth and uniformity in FFPE DNA compared to blood. We also evaluated amplicon-specific differences in coverage depth and observed reduced coverage depth for longer amplicons in FFPE DNA. Furthermore, we showed elevated levels of C$>$T/G$>$A base changes in FFPE specimens compared to blood. This finding is consistent with several studies demonstrating cytosine deamination as the main source of sequence artifact in formalin-fixed specimens. In addition to C$>$T/G$>$A base changes, we observed increased A$>$G/T$>$C in FFPE DNA, although this difference was smaller than C$>$T/G$>$A. Our findings also demonstrated that base changes at the 1-10\%, 10-20\%, and 20-30\% allele frequency ranges were more prevalent in FFPE DNA than blood. This result implies that germline calls should be made at $>$30\% VAF to remove the majority of formalin-induced artifactual changes and somatic mutations. Finally, we showed that sequencing metrics are dependent on storage time of paraffin blocks. Older FFPE blocks were more likely to yield lower efficiency in amplicon enrichment, leading to poorer sequencing results and increased prevalence of artifactual C>T/G>A transitions. Thus, this indicates that if multiple FFPE specimens are available, the specimen with the lower storage time should be selected for molecular analysis.



EGFR-overexpressing cancers are highly aggressive and have a higher tendency to metastasize. Currently, available drugs specifically target the EGFR and elicit high response rates. However, the majority of patients eventually develop progressive disease. The mechanisms through which cancers escape EGFR-targeted therapies remain unclear. Identification of specific molecules that mediate resistance to EGFR-directed treatments will facilitate the development of novel therapies and may improve responses to currently available therapies.

In this study, we measured secreted factors in the media of sensitive and resistant cell lines to identify differentially expressed cytokines that may mediate resistance. Through a combination of ELISAs and mass spectrometry-based assays, we identified cytokine A as being significantly up-regulated in resistant cells. Cytokine A is a major activator of the ABCD signaling cascade (literature citations). The ABCD cascade is a known target of EGFR signaling and is usually blocked in response to EGFR inhibition (literature citations). A previous study demonstrated that exogenous stimulation of ABCD signaling reduces the response to EGFR-targeted drugs (literature citations). This report is consistent with our finding that a major stimulus of ABCD signaling is overexpressed in resistant cells. Based on these data, we propose that hyperactive ABCD signaling is a major mechanism of resistance to EGFR-targeted therapies (Figure XX, schematic of proposed mechanism of resistance). This section will be greatly expanded in a real Discussion section to place your finding in the context of multiple published studies.

In this study, we retrospectively analyzed targeted sequencing data from tumour-blood paired samples of 213 cancer patients to investigate whether potential germline alterations can be accurately identified in FFPE tumours without the use of matched normal samples. Because blood DNA is one of the gold standards for germline testing, we characterized formalin-induced DNA damage in our data to evaluate the quality and usability of FFPE DNA for germline variant calling. Using blood DNA as non-formalin-fixed controls, we compared efficiency in amplicon enrichment and sequencing results of FFPE DNA to blood. We observed lower efficiency in amplicon enrichment in FFPE DNA, which can be caused by reduced template DNA as a result of formalin-induced DNA fragmentation. Although FFPE and blood DNA managed to achieve comparable proportions of on-target read alignments, which are aligned reads used for variant calling, our results showed minor discrepancies in coverage depth and uniformity in FFPE DNA compared to blood. We also evaluated amplicon-specific differences in coverage depth and observed reduced coverage depth for longer amplicons in FFPE DNA.


This result reveals the impact of DNA fragmentation caused by formalin fixation, which gives rise to shorter template DNA that are less amplifiable for longer amplicons.

In addition to cytosine deamination, formaldehyde can react with atmospheric oxygen to result in formic acid, which causes depurinationTo a lesser extent, which could be caused by incorporation of guanines at abasic sites induced by reaction

Although both  and A$>$G/T$>$C were elevated in FFPE specimens compared to the other base transversions, the magnitude of difference was larger for C$>$T/G$>$A than A$>$G/T$>$C (median log\textsubscript{2} fold change: C$>$T/G$>$A = 4.2, A$>$G/T$>$C = 1.6), which further confirms that deamination of cytosine bases is the most frequent form of sequence artifact in FFPE DNA.


Caveats: did not review every single variant
