%% The following is a directive for TeXShop to indicate the main file
%%!TEX root = diss.tex

\chapter{List of Abbreviations}

%This glossary uses the handy \latexpackage{acroynym} package to automatically
%maintain the glossary.  It uses the package's \texttt{printonlyused}
%option to include only those acronyms explicitly referenced in the
%\LaTeX\ source.

% use \acrodef to define an acronym, but no listing
% \acrodef{UI}{user interface}
% \acrodef{UBC}{University of British Columbia}

% The acronym environment will typeset only those acronyms that were
% *actually used* in the course of the document

\begin{acronym}[BCR-ABL1]
\acro{ACCE}{Analytical validity, Clinical validity, Clinical utility, and Ethical, legal and social implications}

\acro{ACMG}{American College of Medical Genetics and Genomics}

\acro{ALK}[\textit{ALK}]{Anaplastic lymphoma kinase gene}

\acro{APC}[\textit{APC}]{Adenomatous polyposis coli gene}

\acro{ASCII}{American Standard Code for Information Interchange}

\acro{BAM}{Binary Alignment$/$Map}

\acro{BAQ}{Base quality score}

\acro{BCR-ABL1}[\textit{BCR-ABL1}]{Breakpoint cluster region and Abelson murine leukemia viral oncogene homolog 1 fusion gene}

\acro{BRAF}[\textit{BRAF}]{B-Raf proto-oncogene}

\acro{BRCA1}[\textit{BRCA1}]{Breast cancer type 1 susceptibility gene}

\acro{BRCA2}[\textit{BRCA2}]{Breast cancer type 2 susceptibility gene}

\acro{BWA}{Burrows-Wheeler aligner}

\acro{BWT}{Burrows-Wheeler transform}

\acro{CDC}{Centers for Disease Control and Prevention}

\acro{CI}{Confidence interval}

\acro{CNV}{Copy number variation}

\acro{COSMIC}{Catalogue of Somatic Mutations in Cancer database}

\acro{CPG}{Cancer predisposing gene}

\acro{CRC}{Colorectal cancer}

\acro{dbSNP}{Single Nucleotide Polymorphism Database}

\acro{dNTP}{Deoxyribonucleotides}

\acro{dTMP}{Deoxythymidine monophosphate}

\acro{DNA}{Deoxyribonucleic acid}

\acro{DPD}{Dihydropyrimidine dehydrogenase enzyme}

\acro{DPYD}[\textit{DPYD}]{Dihydropyrimidine dehydrogenase gene}

\acro{EGFR}[\textit{EGFR}]{Epidermal growth factor receptor gene}

\acro{ExAC}{Exome Aggregation Consortium}

\acro{FAP}{Familial adenomatous polyposis}

\acro{FET}{Fisher's exact test}

\acro{FFPE}{Formalin-fixed Paraffin-embedded}

\acro{GIST}{Gastrointestinal stromal tumour}

\acro{GSH}{Glutathione}

\acro{GSTP}{Glutathione S-transferases}

\acro{GSTP1}[\textit{GSTP1}]{Glutathione S-transferases gene}

\acro{HER2}[\textit{HER2}]{Human epidermal growth factor receptor 2 gene}

\acro{HGP}{Human Genome Project}

\acro{ICGC}{International Cancer Genome Consortium}

\acro{IGV}{Intergrative Genomics Viewer}

\acro{LOH}{Loss of heterozygosity}

\acro{MAPQ}{Mapping quality score}

\acro{MIT}{Massachusetts Institute of Technology}

\acro{MEN2}{Multiple endocrine neoplasia type 2}

\acro{MR}{Methylenetetrahydrofolate reductase enzyme}

\acro{MTHFR}[\textit{MTHFR}]{Methylenetetrahydrofolate reductase gene}

\acro{NHGRI}{National Human Genome Research Institute}

\acro{NGS}{Next-generation sequencing}

\acro{PGx}{Pharmacogenomics}

\acro{PCR}{Polymerase chain reaction}

\acro{PPV}{Positive predictive value}

\acro{RB1}[\textit{RB1}]{Retinoblastoma 1 gene}

\acro{RET}[\textit{RET}]{Ret proto-oncogene}

\acro{SAM}{Sequence Alignment$/$Map}

\acro{TCGA}{The Cancer Genome Atlas}

\acro{TOP}{The OncoPanel Pilot}

\acro{TP53}[\textit{TP53}]{Tumour protein p53 gene}

\acro{TP}{Thymidine phosphorylase enzyme}

\acro{TS}{Thymidylate synthase enzyme}

\acro{TYMP}[\textit{TYMP}]{Thymidine phosphorylase gene}

\acro{TYMS}[\textit{TYMS}]{Thymidylate synthase gene}

\acro{UDG}{Rracil-DNA glycosylase}

\acro{UGT1A}{Uridine diphosphate glycosyltransferase 1 family enzymes}

\acro{UGT1A1}[\textit{UGT1A1}]{Uridine diphosphate glycosyltransferase 1A1 gene}

\acro{VAF}{Variant allele frequency}

\acro{VCF}{Variant Call Format}

\acro{WES}{Whole exome sequencing}

\acro{WGS}{Whole genome sequencing}

\acro{ANOVA}[ANOVA]{Analysis of Variance\acroextra{, a set of
  statistical techniques to identify sources of variability between groups}}
\acro{API}{application programming interface}
\acro{CTAN}{\acroextra{The }Common \TeX\ Archive Network}
\acro{DOI}{Document Object Identifier\acroextra{ (see
    \url{http://doi.org})}}
\acro{GPS}[GPS]{Graduate and Postdoctoral Studies}
\acro{PDF}{Portable Document Format}
\acro{RCS}[RCS]{Revision control system\acroextra{, a software
    tool for tracking changes to a set of files}}
\acro{TLX}[TLX]{Task Load Index\acroextra{, an instrument for gauging
  the subjective mental workload experienced by a human in performing
  a task}}
\acro{UML}{Unified Modelling Language\acroextra{, a visual language
    for modelling the structure of software artefacts}}
\acro{URL}{Unique Resource Locator\acroextra{, used to describe a
    means for obtaining some resource on the world wide web}}
\acro{W3C}[W3C]{\acroextra{the }World Wide Web Consortium\acroextra{,
    the standards body for web technologies}}
\acro{XML}{Extensible Markup Language}
\acro{VUS}{Variant of Unknown Significance}
\end{acronym}

% You can also use \newacro{}{} to only define acronyms
% but without explictly creating a glossary
%
% \newacro{ANOVA}[ANOVA]{Analysis of Variance\acroextra{, a set of
%   statistical techniques to identify sources of variability between groups.}}
% \newacro{API}[API]{application programming interface}
% \newacro{GOMS}[GOMS]{Goals, Operators, Methods, and Selection\acroextra{,
%   a framework for usability analysis.}}
% \newacro{TLX}[TLX]{Task Load Index\acroextra{, an instrument for gauging
%   the subjective mental workload experienced by a human in performing
%   a task.}}
% \newacro{UI}[UI]{user interface}
% \newacro{UML}[UML]{Unified Modelling Language}
% \newacro{W3C}[W3C]{World Wide Web Consortium}
% \newacro{XML}[XML]{Extensible Markup Language}

\endinput
